sonic_platform_base/sonic_xcvr/mem_maps/public/cmis.py

```diff
@@ -52,7 +52,7 @@ def __init__(self, codes):
 
             CodeRegField(consts.CONNECTOR_FIELD, self.getaddr(0x0, 203), self.codes.CONNECTORS),
             CodeRegField(consts.HOST_ELECTRICAL_INTERFACE, self.getaddr(0x0, 86), self.codes.HOST_ELECTRICAL_INTERFACE),
-            CodeRegField(consts.MEDIA_TYPE_FIELD, self.getaddr(0x0, 85), self.codes.MODULE_MEDIA_TYPE),
+            CodeRegField(consts.MEDIA_TYPE_FIELD, self.getaddr(0x0, 85), self.codes.MEDIA_TYPES),
```

Comment:
can you update the codes.MODULE_MEDIA_TYPE in sff8024.py because it actually comes from SFF-8024 spec table. Then this code change is not required.